doc/source/whatsnew/v0.18.0.txt

```diff
@@ -248,6 +248,35 @@ Subtraction by ``Timedelta`` in a ``Series`` by a ``Timestamp`` works (:issue:`1
    ser
    pd.Timestamp('2012-01-01') - ser
 
+
+Signature change for Series.rank and DataFrame.rank and their GroupBy equivalents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The **new** signature is ``.rank(method='average', axis=0, numeric_only=None, na_option='keep', ascending=True, pct=False)``.
+
```

Comment:
@jorisvandenbossche what do you think?

might be more useful to have ``df.rank(1)`` allowed (and use the original ``DataFrame`` signature``, as probably more useful then specifying the ``method``